patch/files/scripts/variables.gd

```diff
@@ -33,12 +33,12 @@ var banditishumanchance = 70.0
 
 ###---Added by Expansion---### Pregnancy Expanded
 var pregduration = 14.0
-var growuptimechild = 2.0
-var growuptimeteen = 4.0
-var growuptimeadult = 6.0
+var growuptimechild = 1.0 #/ralph
```

Comment:
This is interesting now that I think about it, maybe we should move these to expansionsettings.gd?

Reply:
Fixed